src/code/z_debug.c

```diff
@@ -106,60 +108,59 @@ void Regs_Init(void) {
     }
 }
 
-// Called when free movement is active.
-void func_8006375C(s32 arg0, s32 arg1, const char* text) {
+// Function is stubbed. Name is assumed by similarities in signature to `Debug_SetScreenTextColored` and usage.
+void Debug_SetScreenText(u8 x, u8 y, const char* text) {
 }
 
-// Store text during Update, to be drawn later during Draw
-void func_8006376C(u8 x, u8 y, u8 colorIndex, const char* text) {
-    PrintTextBufferEntry* entry;
+void Debug_SetScreenTextColored(u8 x, u8 y, u8 colorIndex, const char* text) {
```

Comment:
I think "Set" can be omitted, the operation is more a "append" anyway, not that the users of this function care that much
`Debug_ScreenText(Colored) (...)` seems fine

Reply:
yeah had this idea but it felt weird without a "verb"
but the verb cant be print or draw cause that happens somewhere else lol

but yeah ill just remove set

Reply:
Might I suggest `AddScreenText` or possibly `AddToScreenText`?

Reply:
`AddScreenText` is okay. its accurate to what its actually doing, but I do kinda think that to someone not aware of the underlying workings, "add screen text" may be confusing. I see what "Add to" is going for, but think its probably even more confusing

I think just saying what it is, that its screen text, is probably fine. but more opinions welcome